index.html

```diff
@@ -59,7 +59,7 @@ <h5 class="pointer">LET'S CONNECT</h5>
         </section>
       </main>
       <!--Project section-->
-      <div id="projects">
+      <section id="projects">
```

Comment:
- [ ] As pointed out  by the previous reviewer [here](https://github.com/Mov305/P1-portofolio/pull/3#discussion_r911434381). Unfortunately, this issue was not solved by the latest changes. You could try adding extra padding to the projects section.